platform/viewer/src/routes/routesUtil.js

```diff
@@ -1,105 +1,141 @@
-import asyncComponent from '../components/AsyncComponent.js';
-
 import OHIF from '@ohif/core';
 const { urlUtil: UrlUtil } = OHIF.utils;
 
-// Dynamic Import Routes (CodeSplitting)
-const IHEInvokeImageDisplay = asyncComponent(() =>
-  import(
-    /* webpackChunkName: "IHEInvokeImageDisplay" */ './IHEInvokeImageDisplay.js'
-  )
-);
-const ViewerRouting = asyncComponent(() =>
-  import(/* webpackChunkName: "ViewerRouting" */ './ViewerRouting.js')
-);
-
-const StudyListRouting = asyncComponent(() =>
-  import(
-    /* webpackChunkName: "StudyListRouting" */ '../studylist/StudyListRouting.js'
-  )
-);
-const StandaloneRouting = asyncComponent(() =>
-  import(/* webpackChunkName: "ConnectedStandaloneRouting" */ '../connectedComponents/ConnectedStandaloneRouting.js')
-);
-const ViewerLocalFileData = asyncComponent(() =>
-  import(
-    /* webpackChunkName: "ViewerLocalFileData" */ '../connectedComponents/ViewerLocalFileData.js'
-  )
-);
-
 const reload = () => window.location.reload();
 
-const ROUTES_DEF = {
-  default: {
-    viewer: {
-      path: '/viewer/:studyInstanceUids',
-      component: ViewerRouting,
-    },
-    standaloneViewer: {
-      path: '/viewer',
-      component: StandaloneRouting,
-    },
-    list: {
-      path: ['/studylist', '/'],
-      component: StudyListRouting,
-      condition: appConfig => {
-        return appConfig.showStudyList !== undefined
-          ? appConfig.showStudyList
-          : true;
-      },
-    },
-    local: {
-      path: '/local',
-      component: ViewerLocalFileData,
-    },
-    IHEInvokeImageDisplay: {
-      path: '/IHEInvokeImageDisplay',
-    },
+const defaultRoutesDefinitions = {
```

Comment:
Similarly, should this be an array of object? Then use each object's `template: string` property to match it with the appropriate registered template?

Reply:
alright :d done

Reply:
yes, done